cfn-resources/project-invitation/cmd/resource/resource.go

```diff
@@ -28,11 +27,11 @@ import (
 	"go.mongodb.org/atlas/mongodbatlas"
 )
 
-var CreateRequiredFields = []string{constants.ProjectID, constants.PubKey, constants.PvtKey, constants.Username}
-var ReadRequiredFields = []string{constants.ProjectID, constants.ID, constants.PubKey, constants.PvtKey}
-var UpdateRequiredFields = []string{constants.ProjectID, constants.ID, constants.PubKey, constants.PvtKey}
-var DeleteRequiredFields = []string{constants.ProjectID, constants.ID, constants.PubKey, constants.PvtKey}
-var ListRequiredFields = []string{constants.PubKey, constants.PubKey}
+var CreateRequiredFields = []string{constants.ProjectID, constants.Username}
+var ReadRequiredFields = []string{constants.ProjectID, constants.ID}
+var UpdateRequiredFields = []string{constants.ProjectID, constants.ID}
+var DeleteRequiredFields = []string{constants.ProjectID, constants.ID}
+var ListRequiredFields = []string{constants.ProjectID}
```

Comment:
Is this a bugfix - ProjectID wasn't required in error before?

Reply:
Yes, only public and private keys were required for list before.
https://www.mongodb.com/docs/atlas/reference/api/project-get-invitations/#request-path-parameters